CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -743,8 +813,40 @@ local function Create3DSettingsIcon(topBarInstance)
 		end
 	end)
 
+
+	local function OnHamburger3DInput(actionName, state, inputObj)
```

Comment:
It also brought a tear to my eye to know such beautiful function names truly existed.